pkg/operator/controller/controller.go

```diff
@@ -57,7 +58,8 @@ import (
 )
 
 const (
-	finalizerName = "operator.cdi.kubevirt.io"
+	cleanupResourcesFinalizer = "operator.cdi.kubevirt.io"
```

Comment:
Any particular reason you renamed this?

Reply:
I am adding a second finalizer, and "finalizerName" is a name that could refer to both. I thought it'd be more clear this way.

Reply:
After changes, this change was not necessary.